tests/platform_tests/test_advanced_reboot.py

```diff
@@ -21,14 +22,15 @@ def test_fast_reboot(request, get_advanced_reboot):
     advancedReboot.runRebootTestcase()
 
 @pytest.mark.usefixtures('get_advanced_reboot')
+@pytest.mark.device_type('vs')
 def test_warm_reboot(request, get_advanced_reboot):
     '''
     Warm reboot test case is run using advacned reboot test fixture
 
     @param request: Spytest commandline argument
     @param get_advanced_reboot: advanced reboot test fixture
     '''
-    advancedReboot = get_advanced_reboot(rebootType='warm-reboot')
+    advancedReboot = get_advanced_reboot(rebootType='warm-reboot', kvmCompatible=True)
```

Comment:
Do we need both `@pytest.mark.device_type('vs')` and `kvmCompatible=True` marker and flags?
Can we not execute only `VS` marked tests by specifying marker in the pytest CLI. In that case `kvmCompatible` flag and the check in `advanced_reboot.py` may be redundant.

Reply:
Thanks for this suggestion! Removed this and checking pytest mark instead.